platform/engineering/internal-tools.md

```diff
@@ -138,68 +138,87 @@ If you don't already have an SSH public key, or you're not sure if you do, compl
 
 ## Configure the SOCKS proxy
 
-These steps assume your SSH keys have been authorized and that you're running on Linux or OSX. There are slightly different commands to connect to the proxy depending on whether you are connected to the VA network or not. You will need to run the SOCKS proxy on your local system whenever you need access to tools on the `vetsgov-internal` domain.
+These steps assume your SSH keys have been authorized and that you're running on Linux, OSX, or have a Unix shell emulator like Git Bash (Windows). There are slightly different commands to connect to the proxy depending on whether you are connected to the VA network or not. You will need to run the SOCKS proxy on your local system whenever you need access to tools on the `*.vfs.va.gov` domain.
 
 1. Save the SSH configuration that you'll need locally to access the remote SSH servers.
-    * Right click on [this link](https://github.com/department-of-veterans-affairs/devops/raw/master/ssh/config) and save as: `~/.ssh/config_va`
-    * Edit `~/.ssh/config` and add this line at the top:
-    `Include ~/.ssh/config_va`
-
-1. Add your SSH key to your local agent with `ssh-add -K ~/.ssh/id_rsa_vetsgov` (for Windows, the command will not require the `-K` flag).
-
-1. Proceed to the instructions below to access the SOCKS proxy.
+    * Click [this link](https://github.com/department-of-veterans-affairs/devops/raw/master/ssh/config) and copy the entire URL within the new tab.
+      1. From within the terminal (Mac) or Git Bash (Windows), run the following command making sure to change `$URL` to the one you now have copied (i.e. `https://raw.githubusercontent.com/department-of-veterans-affairs/devops/master/ssh/config\?token\=ANNHPNCZI2YPEPZFLIAOGQS7GVQSE`): `curl $URL > ~/.ssh/config`. 
+```
+❯ curl https://raw.githubusercontent.com/department-of-veterans-affairs/devops/master/ssh/config\?token\=ANNHPNCZI2YPEPZFLIAOGQS7GVQSE > ~/.ssh/config
+% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  5804  100  5804    0     0  15602      0 --:--:-- --:--:-- --:--:-- 15602
+❯ ls -al ~/.ssh
+total 80
+drwx------   8 jeremybritt  staff    256 Aug  6 11:27 .
+drwxr-xr-x+ 48 jeremybritt  staff   1536 Aug  6 11:50 ..
+-rw-r--r--   1 jeremybritt  staff   5804 Aug  6 11:50 config
+-rw-------@  1 jeremybritt  staff   1766 Oct  9  2019 id_rsa_vetsgov
+-rw-r--r--   1 jeremybritt  staff    381 Nov 12  2019 id_rsa_vetsgov.pub
+```
+
+> If you already have a `~/.ssh/config` file and would like to keep it intact, you can save the file to a different name `~/.ssh/config_va` and edit `~/.ssh/config` to add this line at the top: `Include ~/.ssh/config_va`. Alternatively, you can use the following command within the terminal to automate this process: `grep -qxF 'Include ~/.ssh/config_va' ~/.ssh/config || echo -e "Include ~/.ssh/config_va\n$(cat ~/.ssh/config)" > ~/.ssh/config`.
```

Comment:
idk if it would be better to include this approach first? if the user has an existing `~/.ssh/config` then running the previous command will overwrite it without warning. alternatively use `curl -o` instead of `>` and include info for if `config-1` gets created

Reply:
applied

Reply:
added in https://github.com/department-of-veterans-affairs/va.gov-team/pull/11950/commits/8f987f9deb532082a65de4db43be81e498acec17